databallpy/match.py

```diff
@@ -0,0 +1,199 @@
+                self.tracking_data_provider == other.tracking_data_provider,
+                self.event_data.equals(other.event_data),
+                self.pitch_dimensions == other.pitch_dimensions,
+                self.periods.equals(other.periods),
+                self.frame_rate == other.frame_rate,
+                self.home_team_id == other.home_team_id,
+                self.home_team_name == other.home_team_name,
+                self.home_formation == other.home_formation,
+                self.home_players.equals(other.home_players),
+                self.home_score == other.home_score,
+                self.away_team_id == other.away_team_id,
+                self.away_team_name == other.away_team_name,
+                self.away_formation == other.away_formation,
+                self.away_players.equals(other.away_players),
+                self.away_score == other.away_score,
+            ]
+            return all(res)
+        else:
+            return False
+
+
+def get_match(
+    *,
+    tracking_data_loc: str,
+    tracking_metadata_loc: str,
+    event_data_loc: str,
+    event_metadata_loc: str,
+    tracking_data_provider: str,
+    event_data_provider: str,
+):
+
+    assert tracking_data_provider in [
+        "tracab"
+    ], f"We do not support '{tracking_data_provider}' as tracking data provider yet, "
+    "please open an issue in our Github repository."
+    assert event_data_provider in [
+        "opta"
+    ], f"We do not supper '{event_data_provider}' as event data provider yet, "
+    "please open an issue in our Github repository."
+
+    # Get event data and event metadata
+    if event_data_provider == "opta":
+        event_data, event_metadata = load_opta_event_data(
+            f7_loc=event_metadata_loc, f24_loc=event_data_loc
+        )
+
+    # Get tracking data and tracking metadata
+    if tracking_data_provider == "tracab":
+        tracking_data, tracking_metadata = load_tracab_tracking_data(
+            tracking_data_loc, tracking_metadata_loc
+        )
+
+    # Check if the event data is scaled the right way
+    if not tracking_metadata.pitch_dimensions == event_metadata.pitch_dimensions:
+        x_correction = (
+            tracking_metadata.pitch_dimensions[0] / event_metadata.pitch_dimensions[0]
+        )
+        y_correction = (
+            tracking_metadata.pitch_dimensions[1] / event_metadata.pitch_dimensions[1]
+        )
+        event_data["start_x"] *= x_correction
+        event_data["start_y"] *= y_correction
+
+    # Merge periods
+    merged_periods = pd.concat(
+        (
+            tracking_metadata.periods_frames,
+            event_metadata.periods_frames.drop("period", axis=1),
+        ),
+        axis=1,
+    )
+
+    # Merged player info
+    home_players = tracking_metadata.home_players.merge(
+        event_metadata.home_players, how="outer"
+    )
+    away_players = tracking_metadata.away_players.merge(
```

Comment:
Some players are in the df twice. It seems that this is the case when full_name is different